### PR TITLE
Code for prompt monitoring of DoubleMu3_PFMET50 PFMHT60 path

### PR DESCRIPTION
**Pull request by gkaratha**
code for monitoring the doublemu3_pfmet50_pfmht60 path. It creates efficiencies for met and muon leg. It compiles and runs in CMSSW 921

### CODE REVIEW

**Comment by mtosi** [comment]
please, rebase

File: DQMOffline/Trigger/python/SoftOSMonitor_cff.py [hunk 1]
```diff
@@ -5,17 +5,17 @@
 # HLT_PFMETNoMu90_PFMHTNoMu90_IDTight
 DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring = hltSOSmonitoring.clone()
 DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring.FolderName = cms.string('HLT/SOS/MET/')
-DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DoubleMu3_PFMET50_DZ_PFMHT60_v1")
-DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_v7")
+DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DoubleMu3_PFMET50_DZ_PFMHT60_v*")
```
Comment (mtosi) [inline DQMOffline/Trigger/python/SoftOSMonitor_cff.py]: which is the name of the path ?
in the silvio's google doc I find a slightly different naming :(

**Comment by mtosi** [comment]
I still see SOSMonitor .....

then, where are
HLT_DoubleMu3_DZ_PFMET70_PFMHT70_v*
HLT_DoubleMu3_DZ_PFMET90_PFMHT90_v*
?

**Comment by dmitrijus** [comment]
Where is 93x port?

**Comment by gkaratha** [comment]
Dear dmitrijus,

Sorry for that I though I closed all the spare ones. Mia suggested to modify the TopMonitoring code in an effort to reduce the monitoring scripts. The new PR is here (in 93X) https://github.com/cms-sw/cmssw/pull/19627. Many thanks.

Best regards,
George

**Pull request closed without merging**